<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAccelAsciiImport</class>
 <widget class="QDialog" name="DialogAccelAsciiImport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>PCB file:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPathEdit" name="patheditPCBFile"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Schematic: 1:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Schematic 2:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPathEdit" name="patheditSch2"/>
   </item>
   <item row="1" column="2">
    <widget class="QPathEdit" name="patheditSch1"/>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButtonConvert">
     <property name="text">
      <string>Convert</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPathEdit</class>
   <extends>QWidget</extends>
   <header>qpathedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
